### PR TITLE
challenge/gemma teixido/cv-fullstack

### PR DESCRIPTION
**Pull request by gteixi**
Challenge where I create a **fullstack** website of my CV:

- React
- BEM
- SASS
- Tested
- _Backend_ with Model of diferent sections of my CV
- _Frontend_ with two components 1. Presentation and 2. Details

_Enjoy and feel free to hire me_

### CODE REVIEW

File: gemma-teixido/challenge/cv-fullstack/frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="icon" src="../src/assets/selfie_linkedin.jpg" type="image/x-icon">
```
Comment (IsabelGomez87) [inline gemma-teixido/challenge/cv-fullstack/frontend/public/index.html:11]: Me encanta tu favicon!

**Review by IsabelGomez87 — APPROVED** [review]
Great Job my friend!

File: gemma-teixido/challenge/cv-fullstack/backend/src/model/cvModel.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
```
Comment (IrenePerezAlonso) [inline gemma-teixido/challenge/cv-fullstack/backend/src/model/cvModel.js:1]: So nice this Schema!! :D

**Review by davzlum — APPROVED** [review]
Well done!!

**Review by IrenePerezAlonso — APPROVED** [review]
Very good you CV Gemma :D

**Review by SantiagoMC52 — APPROVED** [review]

**Pull request merged**